siliconcompiler/tools/verilator/verilator.py

```diff
@@ -109,6 +113,8 @@ def runtime_options(chip):
             cmdlist.append('-I' + value)
         for value in chip.find_files('option', 'cmdfile'):
             cmdlist.append('-f ' + value)
+        for value in chip.get('option', 'define'):
+            cmdlist.append('-D' + value)
```

Comment:
Can you switch this to f'-D{value}' instead?

Reply:
Yeah. I was being consistent with the legacy code in that block, but I'll just fix it all